docs/sql-data-sources-json.md

```diff
@@ -155,7 +155,7 @@ Data source options of JSON can be set via:
     <td>read</td>
   </tr>
   <tr>
-    <td><code>allowNumericLeadingZero</code></td>
+    <td><code>allowNumericLeadingZeros</code></td>
```

Comment:
This should fix the https://spark.apache.org/docs/latest/sql-data-sources-json.html#data-source-option. FYI.